docs/source/models/gbd2017/causes/neonatal_other/index.rst

```diff
@@ -7,32 +7,93 @@ Other Neonatal Disorders
-disorders. An overall neonatal disorders “parent” envelope is also estimated, to
-which all neonatal causes are squeezed.
-
-Complete content of "Other neonatal disorders" in YLD Appendix
-++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-
-In addition to the neonatal disorders described above, there are many diverse
-types of neonatal disorders with a range of severities and associated sequelae.
-Because these other neonatal disorders are diverse in their underlying causes
-and risk factors as well as in their associated health outcomes, modelling them
-together in a DisMod-MR model would not produce reliable estimates of prevalence
-or excess mortality. Instead, we calculated the YLDs caused by other neonatal
-disorders directly using a YLD/YLL ratio.
-
-We calculated the ratio of YLDs to YLLs across the specified neonatal disorders
-for which non-fatal outcomes were modelled, using YLL estimates from the GBD
-2017 cause of death (CoD) analysis. We then multiplied this YLD/YLL ratio by the
-YLL estimates for other chronic respiratory diseases from the GBD 2017 CoD
-analysis, providing us with an estimate of the YLDs associated with other
-neonatal disorders.
+Fatal model
++++++++++++
+*Other neonatal disorders* contribute to both fatal and nonfatal health burden. 
+The fatal model is a standard CoDEM model, which runs on vital registration and 
+surveillance data to estimate the proportion of deaths coded to the "neonatal 
+other" ICD codes, listed above.
+
+Within the context of GBD, "neonatal disorders comprises five causes: preterm 
+birth complications, neonatal encephalopathy and birth trauma, neonatal sepsis 
+and other infections, hemolytic disease and neonatal jaundice, and other 
+neonatal disorders. These five CoDEM models are run, in addition to a "parent" 
+model, to which the children models are all squeezed.
+
+Nonfatal estimates
+++++++++++++++
+The nonfatal burden of "other neonatal disorders" is not explicitly modeled by 
+GBD. We here include the complete content of the "Other neonatal disorders" 
+writeup in the 2017 [YLD]_ Appendix:
+
+	In addition to the neonatal disorders described above, there are many diverse
+	types of neonatal disorders with a range of severities and associated sequelae.
+	Because these other neonatal disorders are diverse in their underlying causes
+	and risk factors as well as in their associated health outcomes, modelling them
+	together in a DisMod-MR model would not produce reliable estimates of prevalence
+	or excess mortality. Instead, we calculated the YLDs caused by other neonatal
+	disorders directly using a YLD/YLL ratio.
+
+	We calculated the ratio of YLDs to YLLs across the specified neonatal disorders
+	for which non-fatal outcomes were modelled, using YLL estimates from the GBD
+	2017 cause of death (CoD) analysis. We then multiplied this YLD/YLL ratio by the
+	YLL estimates for other chronic respiratory diseases from the GBD 2017 CoD
```

Comment:
so weird!  why would they use "other chronic respiratory diseases" YLLs?  I bet that is a copy-paste typo, and they meant to write "other neonatal diseases".

Reply:
Oof yes---I'm almost certain when Helena explained this in person she said other neonatal as opposed to other chronic respiratory. I'll double check with her, though. Thanks for the catch!